### PR TITLE
feat: config osv.dev vulnerability alerts

### PR DESCRIPTION
**Pull request by cleverer**

### CODE REVIEW

**Comment by Radu-C-Martin** [comment]
did you go on a tangent? :D

File: default.json [hunk 1]
```diff
@@ -24,5 +24,7 @@
     "enabled": true
   },
   "reviewers": ["team:dev-team"],
-  "labels": ["renovate-bot"]
+  "labels": ["renovate-bot"],
+  "osvVulnerabilityAlerts": true,
+  "dependencyDashboardOSVVulnerabilitySummary": "all"
```
Comment (Radu-C-Martin) [inline default.json:29]: which dependency dashboard?

Comment (cleverer) [reply default.json:29]: The one on renovate bot website

**Review by Radu-C-Martin — APPROVED** [review]
lgtm, if you want to use experimental features :D

**Pull request merged**